test/tracedump.ml

```diff
@@ -22,6 +23,7 @@ let print_error = function
   | `Ambiguous_uri -> prerr_string "More than one provider for a given URI"
 
 let main uri =
+  let _ = Plugins.load() in
```

Comment:
It is better to check error codes here. If there was an error, it will be silently ignored, and a plugin will not be loaded. This will add frustration to users